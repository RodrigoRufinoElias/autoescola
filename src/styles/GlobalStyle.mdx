import { Meta, Canvas } from "@storybook/addon-docs/blocks";
import * as GlobalStyle from "./GlobalStyle.stories";

<Meta title="Styles/GlobalStyle" of={GlobalStyle} />

# Global Style

CSS resets and base stylesheets definition.

```jsx
const App = () => (
  <>
    <GlobalStyle />
    <OtherComponents />
  </>
);
```

## Typography

Basic global typography.

### Headings

All HTML headings, `h1` through `h6`, are available.

<Canvas of={GlobalStyle.headings} />

### Paragraph

Styling form common text elements:

<Canvas of={GlobalStyle.paragraph} />
