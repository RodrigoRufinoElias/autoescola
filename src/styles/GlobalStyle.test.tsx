import { render } from "@testing-library/react";
import GlobalStyle from "./GlobalStyle";

describe("Global Style", () => {
  it("should match snapshot", () => {
    render(<GlobalStyle />);
    expect(document.head).toMatchSnapshot();
  });
});
