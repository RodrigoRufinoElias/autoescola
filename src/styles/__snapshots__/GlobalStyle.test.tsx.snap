// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global Style should match snapshot 1`] = `
<head>
  <style
    data-styled="active"
    data-styled-version="6.1.16"
  />
</head>
`;
